backend/climateconnect_main/utility/general.py

```diff
@@ -10,7 +11,9 @@
 
 
 # source: https://dev.to/ageumatheus/creating-image-from-dataurl-base64-with-pyhton-django-454g
-def get_image_from_data_url(data_url, resize=False, base_width=500):
+def get_image_from_data_url(
+    data_url, resize=False, base_width=500
+) -> tuple[ContentFile, tuple[str, Any]]:
```

Comment:
Here I'm getting  this error when trying to run the backend: 
```
File "/home/chris/climateconnect_env/climateconnect/backend/climateconnect_main/utility/general.py", line 16, in <module>
    ) -> tuple[ContentFile, tuple[str, Any]]:
TypeError: 'type' object is not subscriptable
```

Reply:
I'm getting the following hint from vscode to fix this: `Subscript for class "list" will generate runtime exception; enclose type annotation in quotes`

Reply:
Same thing in organization/utility/project.py at this position: `def get_project_helpful_connections(
    project: Project, language_code: str
) -> Optional[list[str]]:`

Reply:
```suggestion
) ->"tuple[ContentFile, tuple[str, Any]]":
```

Reply:
I'm not sure how much effort and risk a python minor version upgrade is. Would you recommend to go for the upgrade or is this a bigger project? What are your thoughts on this @ddhanesha ?